draft-ietf-tls-tlsflags.xml

```diff
@@ -150,6 +150,13 @@
             <t> A server MAY indicate support for flag-type features in a flags extension of NewSessionTicket (NST).
                 This message has no client-side response.</t>
             <t> In summary, unsolicited flags may appear only in ClientHello, CertificateRequest, and NewSessionTicket.</t>
+            <t> An implementation that receives an invalid tls_flags extension MUST terminate the TLS handshake with a
+                fatal illegal_parameter alert. Such invalid tls_flags extensions include: <list style="symbols">
+                <t> A zero-length extension</t>
```

Comment:
If we want this to be invalid, we can put it in the TLS presentation language. Change `FlagExtensions` to:

```
      struct {
         opaque flags<1..255>;
      } FlagExtensions;
```